blog/2021-08-12-state_class_total.md

```diff
@@ -0,0 +1,74 @@
+---
+author: Erik Montnémery
+authorURL: https://github.com/emontnemery
+title: "New sensor state classes: total and total_increasing"
+---
+
+Two new state classes, `STATE_CLASS_TOTAL` and `STATE_CLASS_TOTAL_INCREASING` have been
+added. In addition, it is no longer mandatory to set the `last_reset` attribute in order
+for `sum` statistics to be generated. The driver for the changes is to make it easier to
+integrate with utility meters etc.
+
+### State classes
+
+There are now 3 defined state classes:
+- `STATE_CLASS_MEASUREMENT`, the state represents a measurement in present time, for 
+   example a temperature, electric power etc.
+- `STATE_CLASS_TOTAL`, the state represents a total amount which can both increase and
+   decrease, e.g. the value of a stock portfolio
+- `STATE_CLASS_TOTAL_INCREASING`, a monotonically increasing total, e.g. an amount of
+   consumed gas, water or energy
+
+#### STATE_CLASS_TOTAL
+If the sensor's state class is `STATE_CLASS_TOTAL`, the last_reset attribute can
+optionally be set to gain manual control of meter cycles; each time last_reset changes
+the corresponding value is used as the zero-point when calculating `sum` statistics.
+If last_reset is not set, the sensor's value when it was first added is used as the
```

Comment:
"first added" -> when looking at the examples it looks like it uses the last state as the zero-point, not the first state.